spec/shadow/index.html

```diff
@@ -1139,6 +1141,27 @@
           </li>
         </ol>
 
+        <p>Likewise, the <a>sequential focus navigation order</a> for <a>distributed nodes</a>
+          under a given <a>slot</a> <var>A</var> <strong>must</strong> be inserted into
+          the <a>sequential focus navigation order</a> of the containing <a>shadow tree</a>'s
+          <a>sequential focus navigation order</a> as follows:</p>
+        <ol>
+          <li>The <a>sequential focus navigation order</a> for its <a>distributed nodes</a>
+            <strong>must</strong> be skipped if A is assigned <a>tabindex</a> value -1.
+          <li>If <var>A</var> is not given <a>tabindex</a>,
+            The <a>sequential focus navigation order</a> for its <a>distributed nodes</a>
+            <strong>must</strong> be inserted in place of <var>A</var> as if <var>A</var>
+            were assigned the <a>tabindex</a> value 0 for determining its position.</li>
+          <li>Otherwise the <a>sequential focus navigation order</a> for its
+            <a>distributed nodes</a> <strong>must</strong> be inserted in place of
+            <var>A</var> according to its <a>tabindex</a> value.</li>
+        </ol>
+        <p class="note">When a <a>slot</a> <var>A</var> is assigned to another
```

Comment:
We do not need to mention this. Please try to have a general rule which covers this case.